osu.Game/Graphics/Containers/ScalingContainer.cs

```diff
@@ -90,6 +91,12 @@ public partial class ScalingDrawSizePreservingFillContainer : DrawSizePreserving
             private readonly bool applyUIScale;
             private Bindable<float> uiScale;
 
+            private static readonly Vector2 base_target_draw_size = new Vector2(1024, 768);
```

Comment:
Where does this draw size come from?

I am also wondering if this is something that could be thrown into a configuration file or class to make it easier to maintain/change.

Reply:
> I am also wondering if this is something that could be thrown into a configuration file or class to make it easier to maintain/change.

should not need changing